cluster-autoscaler/utils/taints/taints.go

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package taints
```

Comment:
This is not strictly related to this PR, but if you're adding this anyway it would be nice to also extrace taint-related logic from cluster-autoscaler/core/utils/utils.go (consts + code filtering out taints) and put it here.

Reply:
I tried to do so, it's very tangle so moving a lot of code, would you still me to extract this there ?

Reply:
I though to just move https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/core/utils/taint_key_set.go (pretty much cut-paste), const definitions in https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/core/utils/utils.go#L42 and extract the part of sanitizeNodeInfo that deals with taints (https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/core/utils/utils.go#L263) to a separate method. At least that's the stuff I spotted in 5 minute look at the code - I may have missed something else. It seemed like this should be easy to move and it would be nice to get all taint consts and basic taint filtering logic in one place (also it would remove the need to import from core).

Feel free to ignore this comment if it's more messy than I think it is.

Reply:
@MaciekPytel I've done the requested change, indeed with just extracting some part of the code it's trivial.
Let me know if the current PR needs some additional changes.